src/main/java/io/github/sskorol/assertions/LoginPageAssert.java

```diff
@@ -1,29 +1,30 @@
 package io.github.sskorol.assertions;
 
-import io.github.sskorol.model.Account;
+import io.github.sskorol.pages.LoginPage;
 import io.qameta.allure.Step;
 import org.assertj.core.api.AbstractAssert;
 import org.assertj.core.util.Objects;
 
 /**
  * A simple model class for Login Page assert processing.
  */
-public class LoginPageAssert extends AbstractAssert<LoginPageAssert, Account> {
+public class LoginPageAssert extends AbstractAssert<LoginPageAssert, LoginPage> {
 
     private static final String ERROR_MESSAGE_TEMPLATE = "\nExpecting <%s> of: \n<%s>\nto be: <%s>\nbut was: <%s>\n";
 
-    public LoginPageAssert(final Account account) {
-        super(account, LoginPageAssert.class);
+    public LoginPageAssert(final LoginPage actual) {
+        super(actual, LoginPageAssert.class);
     }
 
-    @Step("Verify that the login status is equal to \"{loginStatus}\".")
-    public LoginPageAssert hasLoginStatus(final String loginStatus) {
+    @Step("Verify that the login status is equal to \"{value}\".")
+
+    public LoginPageAssert hasLoginStatus(final String value) {
         isNotNull();
 
-        final String status = actual.getLoginStatus();
+        final String loginStatus = actual.getLoginStatus();
 
-        if (!Objects.areEqual(status, loginStatus)) {
-            failWithMessage(ERROR_MESSAGE_TEMPLATE, "Status name", actual, loginStatus, status);
+        if (!Objects.areEqual(loginStatus, value)) {
+            failWithMessage(ERROR_MESSAGE_TEMPLATE, "Status name", actual, loginStatus);
```

Comment:
Need to pass 4 strings into ERROR_MESSAGE_TEMPLATE. You missed the expected status ("value").